README.md

```diff
@@ -1 +1,61 @@
-# capstone
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Designing a Responsive page Using HTML&CSS
+
+
+> Description the project.
+
+
+## Built With
+
+- html&css
+- no Frameworks
+- Flexbox
+- linter
+
```

Comment:
- [ ] I can't find any live demo link working in your readme. According to the project, the requirement you should use GitHub Pages to deploy your website and make sure that all pages, links, and images work as expected then add the link in the README. Please fix it :+1: